src/Components/Category/Category.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'react-bootstrap/Image';
+
+import './Category.scss';
+import '../../Styles/fontello/css/fontello.css';
```

Comment:
To jest raczej globalny import, który powinien być zaimportowany gdzieś w roocie aplikacji, bo zapewne fontello będzie używane jeszcze w kilku miejscach, a nie tylko tutaj.